report-processing-service/package.json

```diff
@@ -33,10 +36,14 @@
   },
   "dependencies": {
     "amqplib": "^0.5.2",
+    "fs": "0.0.1-security",
```

Comment:
what's the reason for this package?
https://www.npmjs.com/package/fs

there is native module `fs` to work with the file system